src/backend/distributed/commands/table.c

```diff
@@ -1589,11 +1599,10 @@ PreprocessAlterTableStmt(Node *node, const char *alterTableCommand,
 	DDLJob *ddlJob = palloc0(sizeof(DDLJob));
 	ObjectAddressSet(ddlJob->targetObjectAddress, RelationRelationId, leftRelationId);
 
-	const char *sqlForTaskList = alterTableCommand;
```

Comment:
not sure why we had this variable, we should be using the same command on shards and shell table unless otherwise specified (e.g. via useInitialDDLCommandString)